f/radio/fn_radioCheckChannels.sqf

```diff
@@ -0,0 +1,86 @@
+// F3 - Folk ARPS Long-Range Radio Module
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+
+/* ========================
+This function checks what channels a player should have access to. It's called by event handlers set up by fn_radioAddHandlers.sqf.
+=========================== */
+
+// Clients wait for the server to finish with setup
+waitUntil {!(isNil "f_var_radioChannelsUnified")};
+
+_splitMode = f_var_radioSplitMode;
+
+_channelsToAddListen = [];
+_channelsToAddTalk = [];
+_isVehicle = false;
```

Comment:
all of the variables should probably be marked as `private`

Reply:
`_isVehicle` isn't used?

Reply:
Fixed

Reply:
_isVehicle removed; was part of a different system that got halfway before being replaced.